.github/workflows/ci.yml

```diff
@@ -83,7 +85,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: ['3.9', '3.10', '3.11']
+        python-version: ${{ env.SUPPORTED_PYTHON_VERSIONS }}
         should-release:
```

Comment:
This is what has to be reverted

Reply:
Have a look at the error: https://github.com/ansys/pyfluent/actions/runs/9210539031/workflow

Reply:
Thank you.